client/src/main.js

```diff
@@ -9,7 +9,7 @@ import io from "socket.io-client";
 Vue.config.productionTip = false;
 
 // ATTN: Uncomment accordingly for local/remote dev
-const ENDPOINT = "https://www.toonin.ml:8443/";
+const ENDPOINT = "https://www.toonin.ml:443/";
```

Comment:
Shouldn't this still be 8443. This is the Endpoint to the signalling server

Reply:
everything is on 443 now. It's one node server.

Reply:
Makes sense. Can you setup temporary redirect from 8443 to 443 for backend requests until we get the updated extension on webstore?